src/main/java/com/skynest/tests/files/EndpointConstants.java

```diff
@@ -1,6 +1,6 @@
 package com.skynest.tests.files;
 
 public class EndpointConstants {
-    public static final String url = "http://localhost:8080";
-    public static final String registerUrl="/users/register";
+    public static final String URL = "http://localhost:8080";
+    public static final String REGISTER_URL = "/users/register";
```

Comment:
technically speaking, this is not a url but a path; so "REGISTER_PATH" might be more appropriate